ui/src/main/webapp/src/app/shared/validators/project-name-not-exists.validator.ts

```diff
@@ -21,6 +24,10 @@ export class ProjectNameNotExistsValidator implements Validator {
         let projectService = this.projectService;
 
         return new Promise(resolve => {
+            if (this.editMode) {
+                resolve(null);
```

Comment:
Oh, yeah, editMode will allow duplicate titles in edit if you pick a title from a different project. :)

I have updated it to take the project model and do an id comparison. I think this is more valid.